packages/components/src/snackbar/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { flow, noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { RawHTML, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '../';
+
+const stopPropagation = ( event ) => event.stopPropagation();
+const NOTICE_TIMEOUT = 10000;
+
+function Snackbar( {
+	className,
+	status,
+	children,
+	actions = [],
+	__unstableHTML,
+	onRemove = noop,
+} ) {
+	useEffect( () => {
+		// This rule doesn't account yet for React Hooks
+		// eslint-disable-next-line @wordpress/react-no-unsafe-timeout
```

Comment:
This rule doesn't account for React Hooks. It should be possible to use and cancel timeouts in useEffect or other hooks.

Reply:
Tracked here https://github.com/WordPress/gutenberg/issues/15622